### ISSUE TITLE
isar.dev is down/dysfunctional

### ISSUE THREAD

**Issue by AKushWarrior** [isar/isar-core#2]
Just thought you might want to know. Not sure if it's intended behavior ofc ;)

**Comment by leisim** [comment]
Yes, the website doesn't exist yet. I'll use this issue to track it's progress :+1:
